WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -169,6 +174,18 @@ private IEnumerable<Money> DecomposeAmounts(FeeRate feeRate)
 			yield return realAmountCredentialValues.Zip(realVsizeCredentialValues, outputValues, (a, v, o) => (a, v, o));
 		}
 
+		private async Task<IEnumerable<(Money Amount, Credential[] RealAmountCredentials, Credential[] RealVsizeCredentials)>> ReissuancesAsync(BobClient bobClient, CancellationToken cancellationToken)
+		{
+			int outputVsize = Constants.OutputSizeInBytes;
```

Comment:
These should just be an `IEnumerable<ulong>` that gets passed through, there shouldn't be a need to create them. I think this is a side effect of the `ArenaClient` calling them that in the parameters, script sizes as a concept only apply to input and output registrations, reissuances only care about whether two lists (one per type) of k credentials each add up to exactly the same value.